sklearn/linear_model/_coordinate_descent.py

```diff
@@ -802,7 +804,7 @@ def fit(self, X, y, sample_weight=None, check_input=True):
             # enet_path therefore sets alpha = n_samples * alpha
             # With sw, enet_path should set alpha = sum(sw) * alpha
             # Therefore, we rescale alpha = sum(sw) / n_samples * alpha
-            # Note: As we rescaled sample_weights to sum up to n_samples,
+            # Note: As we rescaled sample_weight to sum up to n_samples,
             #       we don't need this
             # alpha *= np.sum(sample_weight) / n_samples
```

Comment:
This looks like a duplicated comment (see bellow).
Could we factor them in this PR?